.ci/1.16/develop-with-kafka/workspace.yml

```diff
@@ -0,0 +1,14 @@
+workspace('ci-go-sample'):
+  description: generated local workspace for ci-go-sample.
+  harness: inviqa/go
+
+attributes:
+  app:
+    build: develop
+    services: [postgres]
```

Comment:
this is a mistake, but should have failed the Jenkins pipeline, will investigate

Reply:
oops, I had not added it to the Jenkinsfile which is why it wasn't failing the pipeline, updated now and fixed this test file